src/model_scoring_functions.R

```diff
@@ -143,17 +120,18 @@ calc_energy_scores <- function(targets, df_model_output){
       df_obs <- subset(targets, target_date == as.Date(day) & location == loc) |>
         group_by(clade)
 
-      # If there is no such observation, skip to next day
-      if( nrow(df_obs) == 0 ){
-        next
+      # Scored location/date pair?
+      scored <- df_obs$scored[1]
+      if(is.na(scored)){
+        scored <- FALSE
```

Comment:
not needed now that data has been coalesced? maybe replace with a warning  or error since it would be a problem if it were an NA?